### PR TITLE
Rename master branch to main; fix links to upstream docs

### PR DESCRIPTION
**Pull request by bgilbert**
We can't change the version `master` to `main` in `antora.yml` because `master` is special-cased to produce non-versioned URLs.

Hold until cutover.

### CODE REVIEW

**Comment by travier** [comment]
We will probably have to update the branch in https://pagure.io/fedora-docs/docs-fp-o/blob/prod/f/site.yml#_100 too

**Comment by dustymabe** [comment]
> We will probably have to update the branch in https://pagure.io/fedora-docs/docs-fp-o/blob/prod/f/site.yml#_100 too

yep :) - proposed that change here: https://pagure.io/fedora-docs/docs-fp-o/pull-request/159

File: modules/ROOT/pages/debugging-with-toolbox.adoc [hunk 1]
```diff
@@ -65,4 +65,4 @@ toolbox rm --force fedora-toolbox-32
 
 NOTE: The toolbox utility allows you to create toolboxes with your custom
 images. You can find more details in the
-https://github.com/containers/toolbox/tree/master/doc[toolbox manpages].
+https://github.com/containers/toolbox/tree/main/doc[toolbox manpages].
```
Comment (dustymabe) [inline modules/ROOT/pages/debugging-with-toolbox.adoc:68]: Nice. Already renamed upstream!

File: modules/ROOT/pages/running-containers.adoc [hunk 2]
```diff
@@ -69,4 +69,4 @@ systemd:
 ----
 
 === For more information
-See the https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/container.md#docker[etcd documentation] for more information on running etcd in containers and how to set up multi-node etcd.
+See the https://etcd.io/docs/v3.4/op-guide/container/#docker[etcd documentation] for more information on running etcd in containers and how to set up multi-node etcd.
```
Comment (dustymabe) [inline modules/ROOT/pages/running-containers.adoc:72]: Is there an uversionsed version of these docs (i.e. latest). If there is, should we use that instead?

**Review by dustymabe — APPROVED** [review]
LGTM

File: modules/ROOT/pages/running-containers.adoc (hunk 2 above)
Comment (bgilbert) [reply modules/ROOT/pages/running-containers.adoc]: In fact, there is.  Updated.

**Pull request merged**